proposals/lip-0052.md

```diff
@@ -30,329 +31,345 @@ Therefore, providing a unified module to handle, transfer and modify NFTs is a n
 
-| Name                                  | Type    | Value           |
-|---------------------------------------|---------|-----------------|
-| **Interoperability Constants**        |         |                 |
-| `MIN_RETURN_FEE`                      | uint64  | 1000            |
-| **NFT Module Constants**              |         |                 |
-| `MODULE_ID_NFT`                       | uint32  | TBD             |
-| `COMMAND_ID_TRANSFER`                 | uint32  | 0               |
-| `COMMAND_ID_CROSS_CHAIN_TRANSFER`     | uint32  | 1               |
-| `CROSS_CHAIN_COMMAND_ID_TRANSFER`     | uint32  | 0               |
-| `CHAIN_ID_ALIAS_NATIVE`               | uint32  | 0               |
-| `NFT_NOT_LOCKED`                      | uint32  | `MODULE_ID_NFT` |
-| `MAX_BYTE_LENGTH_ATTRIBUTES`          | uint32  | 9 * 1024        |
-| `CCM_STATUS_OK`                       | uint32  | 0               |
-| `CCM_STATUS_NFT_NOT_SUPPORTED`        | uint32  | 64              |
-| `CCM_STATUS_NFT_PROTOCOL_VIOLATION`   | uint32  | 65              |
-| **Store Constants**                   |         |                 |
-| `STORE_PREFIX_NFT`                    | bytes   | 0x00 00         |
-| `STORE_PREFIX_USER`                   | bytes   | 0x80 00         |
-| `STORE_PREFIX_COLLECTION`             | bytes   | 0xc0 00         |
-| `STORE_PREFIX_AVAILABLE_COLLECTION`   | bytes   | 0xd0 00         |
-| `STORE_KEY_LENGTH_NFT`                | uint32  | 16              |
-| **General Constants**                 |         |                 |
-| `EMPTY_BYTES`                         | bytes   | ""              |
-| `ADDRESS_LENGTH`                      | uint32  | 20              |
-
-#### uint32be Function
-
-The function `uint32be(x)` returns the big endian uint32 serialization of an integer `x`, with `0 <= x < 2^32`. This serialization is always 4 bytes long.
+| Name                                | Type   | Value                   |
+| ----------------------------------- | ------ | ----------------------- |
+| **Interoperability Constants**      |        |                         |
+| `MAINCHAIN_ID`                      | bytes  | 0x 00 00 00 00          |
+| **NFT Module Constants**            |        |                         |
+| `MODULE_NAME_NFT`                   | string | "nft"                   |
+| `COMMAND_NAME_TRANSFER`             | string | "transfer"              |
+| `COMMAND_NAME_CROSS_CHAIN_TRANSFER` | string | "transferCrossChain"    |
+| `CROSS_CHAIN_COMMAND_NAME_TRANSFER` | string | TBD                     |
+| `CCM_STATUS_OK`                     | uint32 | 0                       |
+| `CCM_STATUS_NFT_NOT_SUPPORTED`      | uint32 | 64                      |
+| `CCM_STATUS_PROTOCOL_VIOLATION`     | uint32 | 65                      |
+| `NFT_NOT_LOCKED`                    | string | `MODULE_NAME_NFT`       |
+| `MAX_BYTE_LENGTH_ATTRIBUTES`        | uint32 | 9 \* 1024               |
```

Comment:
Why did we choose 9kb here? Notice that previously 9kb was the limit for the whole array, while now we are limiting only one entry, so this should probably be smaller (or we should rather check the whole array size)